### PR TITLE
Clean up code, fix link URL, and add badge to tweets

### PR DESCRIPTION
**Pull request by meltingice**
Did a couple of things in this PR:
- Tweets now include the story type in front of the title, if present.
- Links are now formatted to remove the `api-` part of the link.
- Included `StoryFormatter` as a module in Story instead of being standalone.
- No longer need to instantiate `Stories`.
- Moved Twitter client into module and out of global variable.

All the changes have been manually tested against a test twitter account. Cheers!

### CODE REVIEW

File: lib/designernews/story.rb [hunk 1]
```diff
@@ -12,5 +14,22 @@ def new?
       created_at = DateTime.parse(@created_at).new_offset('-7').to_time
       created_at > Timeline.most_recent.created_at
     end
+
+    def url
+      @url.gsub(/api-/, '')
```
Comment (cesarfigueroa) [inline lib/designernews/story.rb:19]: @meltingice Will the API ever return URLs without the prefix?

**Comment by cesarfigueroa** [comment]
@meltingice, this is amazing! Thank you so much for your contribution. I’ll merge the changes as soon as I can.

File: lib/designernews/story.rb (hunk 1 above)
Comment (meltingice) [inline lib/designernews/story.rb:19]: Yeah we might change it so it uses the normal `news.layervault.com` URL in the future. When that happens though, it won't break the bot. Can remove the code after that time whenever.

Comment (cesarfigueroa) [inline lib/designernews/story.rb:19]: @meltingice Sounds great! I was annoyed by the prefixed URLs as well.

File: lib/designernews/story.rb [hunk 2]
```diff
@@ -12,5 +14,22 @@ def new?
       created_at = DateTime.parse(@created_at).new_offset('-7').to_time
```
Comment (cesarfigueroa) [inline lib/designernews/story.rb:14]: @meltingice Any plans to add a time zone to the `created_at` field? I don’t like relying on this solution.

Comment (meltingice) [inline lib/designernews/story.rb:14]: I was actually playing around with this, but was afraid to change it for fear of messing up the Twitter account. For what it's worth, DN timestamps are and will always be in UTC.

**Pull request merged**